cloud/helm-chart/pom.xml

```diff
@@ -44,7 +44,8 @@
               <tpl>SCRIPT_STYLE</tpl>
             </mapping>
             <excludes>
-              <exclude>src/main/helm/artifacthub-repo.yml</exclude>
+              <exclude>src/main/helm/zilla/artifacthub-repo.yml</exclude>
+              <exclude>src/main/helm/zilla/**</exclude>
```

Comment:
The second one is sufficient to encompass the first one, right?

Reply:
Yep, true. Here's the fix: https://github.com/aklivity/zilla/pull/912